vulnerability-report.md

```diff
@@ -0,0 +1,108 @@
+#Vulnerability Report
+
+**A1 - Injection**
+
+Since the upload function takes in a request argument, doing some validation on the request.user.is_authenticated, request.user.username, and request.user.id objects would probably be pretty helpful. Alienation and cleansing of the entered data would also be helpful. Parsing it for terms, especially SQL query calls (FROM, SELECT, DROP, *, TABLE) would also be helpful to securing the form.
```

Comment:
ORM's exist for a good reason.  This is one of those good reasons.  Whenever you have access to an abstraction layer for your database, use it.  Raw SQL statements are bad.